dataGenerator.js

```diff
@@ -88,182 +52,36 @@ while (numberOfReviews < numberToWrite) {
   for (var i = 0; i <= randomReviewAmount; i++) {
```

Comment:
for line 47 ( for whatever reason I can not comment on it): why do you need product_id with the starting index of 1? If it can be indexed at 0, that may be cleaner code.